### PR TITLE
[Handler] Keep last events

### PR DESCRIPTION
**Pull request by tjoyal**
Extraction from #85 

Record on the `Handler` model the last 10 events.

I'm now wondering if this is "good enough" or if we should look at aggregating everything on the `Rule` so we can capture filters. 

Thoughts?

### CODE REVIEW

File: lib/handlers/base.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def setting(name, options = {})
         settings[name] = options
 
         define_method(name) do
-          parser.parse(@settings[name])
+          parser.parse(@settings[name.to_s])
```
Comment (tjoyal) [inline lib/handlers/base.rb:32]: I need to have a closer look why retrieving a model from SQL I get a `HashWithIndiferentAccess`  but not using fixtures where I retrieve a `Hash`.

Comment (tjoyal) [reply lib/handlers/base.rb:32]:
```
 Handler.first.attributes_before_type_cast["settings"]
  Handler Load (1.1ms)  SELECT "handlers".* FROM "handlers" ORDER BY "handlers"."id" ASC LIMIT $1  [["LIMIT", 1]]
=> "--- !ruby/hash:ActiveSupport::HashWithIndifferentAccess\nslack_token: xoxp-111111111111111\nchannel: \"#aaaaaaa\"\nmessage: The user {{ first_name }} {{ last_name }}.\n"                                                                               
```

I'm a little bit puzzled here....

**Review by christianblais — APPROVED** [review]
C'est quoi le fichier identifier.sqlite?

Sinon, je pense que c'est bon. Pour ce qui est de filter, on pas pourra rajouter leur propre colonne aussi plutôt que d'agréger tout sur rule.

**Comment by tjoyal** [comment]
Solved via https://github.com/christianblais/triggerify/pull/116

**Pull request closed without merging**